website/addons/s3/provider.py

```diff
@@ -0,0 +1,21 @@
+from website.addons.s3.serializer import S3Serializer
+
+class S3Provider(object):
+    """An alternative to `ExternalProvider` not tied to OAuth"""
+
+    name = 'Amazon S3'
+    short_name = 's3'
+    provider_name = 's3'
```

Comment:
I thought it was required by one of the mixins, but it looks like it's not. I'll remove it.